dev/io.openliberty.servlet.internal.6.0.factories/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
```

Comment:
Servlet 6.0 requires Java 11.  Im not sure if this eclipse jdt can be at 1.8?

Reply:
This is not a concern. This particular bundle did not require anything from Java 11 and Open Liberty's implementation will run with lots of bundles that are compiled at Java 8 because it shares those between EE 10 and EE 8 and so forth.  In this case, we could move it up to Java 11 because this bundle isn't shared, but this is necessary.